Sell screen updates

lmillet3804 opened the pull request.

lmillet3804 committed:
finished sell screens except for api backend integration

lmillet3804 committed:
pull down to refresh on sell screen

TheGoldenPatrik1 commented on app/sell/index.js:
```diff
@@ -47,6 +48,14 @@ const Sell = () => {
 
+    const [refreshing, setRefreshing] = useState(false)
+
+    const onRefresh = useCallback(() => {
```
Would it be possible to make the refresh event also reset the 60 second timer?

TheGoldenPatrik1 replied:
My last comment might have been unclear, but we don't want manually refreshing to **stop** the periodic refresh, only to **reset** it. So maybe make this code a separate function:
```
        setTimeInterval(
            setInterval(() => {
                dispatch(getOrders)
            }, 60000)
        )
```
that gets called both within the useEffect hook and at the end of onRefresh?

lmillet3804 replied:
Okay I fixed it now. I think I was just unclear about what clearInterval() did

TheGoldenPatrik1 commented on app/sell/success.js:
```diff
@@ -0,0 +1,51 @@
+            <View style={styles.success}>
+                <Text style={styles.text}>
+                    Thank you for using MealMatch. Your funds will be released
+                    in the next 2 hours if there are no buyer complaints. {'\n'}
```
Let's change this wording to "Thank you for using MealMatch. If there are no buyer complaints in the next two hours, your funds will be released sometime after that."

lmillet3804 committed:
addressed PR comments

- pull down to refresh resets timer
- confirm order redux is ready for backend api integration
- success page text is changed

lmillet3804 committed:
fixed refresh timer issues
